min/config.php

```diff
@@ -170,7 +166,7 @@
  * To minify all files, set this option to null. You could also specify your
  * own pattern that is matched against the filename.
  */
-$min_serveOptions['minApp']['noMinPattern'] = null;
+//$min_serveOptions['minApp']['noMinPattern'] = '@[-\\.]min\\.(?:js|css)$@i';
```

Comment:
Double anti-slash? Shouldn't be a single one? (yeah I saw it's commented)

Reply:
code copied directly from the lib, we don't use it so /shrug